### PR TITLE
chore: change default api url value

### PR DESCRIPTION
**Pull request by BayBreezy**
## Types of changes

- [x] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Description

When using any node version above 16, the cookie was not being persisted across browser refreshes. Changing "http://localhost:1337" -> "http://127.0.0.1:1337" fixed this issue. Refer to this issue: https://github.com/nuxt-modules/strapi/issues/229

Resolves: #229 

## Checklist:

- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes (if not applicable, please state why)

### CODE REVIEW

File: src/module.ts [hunk 1]
```diff
@@ -74,15 +74,15 @@ export default defineNuxtModule<ModuleOptions>({
...
+    url: process.env.STRAPI_URL || 'http://127.0.0.1:1337',
     prefix: '/api',
     version: 'v4',
     cookie: {},
     auth: {},
     cookieName: 'strapi_jwt',
     devtools: false
   },
-  setup (options, nuxt) {
+  setup(options, nuxt) {
```
Comment (benjamincanac) [inline src/module.ts:85]:
```suggestion
  setup (options, nuxt) {
```

**Review by benjamincanac — APPROVED** [review]

**Comment by BayBreezy** [comment]
> As a note, when using an oauth provider, this does imply changing the Callback URL and setting the `HOST` to `127.0.0.1` in your Strapi config.

Thanks. I think ppl would know to do that, I may be wrong tho..
Do you think it should be mentioned in the docs?

**Comment by HenrijsS** [comment]
Hey.

I changed the host already to 127.0.0.1, because my cookies were getting cleared, but the issue persists.

The URL is getting overwritten, then rewritten back from localhost to 127.0.0.1

Video: [https://share.cleanshot.com/JphHZDtB](https://share.cleanshot.com/JphHZDtB)

Also got a hydration mismatch.

**Comment by benjamincanac** [comment]
Reverted this in favor of: https://github.com/nuxt-modules/strapi/commit/93b8bb8d15bb59d54179c9e8bb0291955078212c

**Comment by BayBreezy** [comment]
Not yet lol.
But it seems like a better option

**Comment by BayBreezy** [comment]
@benjamincanac Yes, it works. I update strapi to 4.15.4 & Nuxt to 3.8.2 on Node v18.18.1
The URL in my `.env` is `STRAPI_URL=http://localhost:1337` and it keeps the cookie in place event after a refresh. So the user stays logged in.
Good job 👍🏽

**Comment by rrubio** [comment]
@BayBreezy  just checking. I've upgraded my strapi instance and nuxt to match yours. I'm still having the same issue. On your .env file, under a dev, prod environment that url will need to be what ever the strapi hosting URL is. I'm assuming you have changed that and tested ok?

**Pull request merged**